virttest/qemu_devices/qcontainer.py

```diff
@@ -1634,6 +1645,7 @@ def images_define_by_variables(self, name, filename, pci_bus, index=None,
         :param image_encryption: ImageEncryption object for image
         :param image_access: The logical image access information object
         :param external_data_file: external data file for qcow2 image
+        :param image_filter_throttle: The throttle group
```

Comment:
```suggestion
        :param image_throttle_group: The throttle group for image
```
I think we could just use `image_throttle_group` according the usage, such as ` -blockdev driver=throttle,throttle-group=xxx`, don't need to declare the prefix `filter_`

Reply:
updated